Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -321,11 +323,13 @@ namespace Babylon
         return false;
     }
 
-    void UpdateXRView(void* windowPtr)
+#if defined(__APPLE__) || defined(ANDROID)
```

Comment:
Why is this needed? Don't we have a no-op impl on other platforms?

Reply:
Other platforms have a type that isn't convertible to `void*` which is what the underlying XR.h currently takes.